server/src/test/java/org/elasticsearch/indices/recovery/RecoverySourceHandlerTests.java

```diff
@@ -333,9 +333,7 @@ public void close() throws IOException {
 
 
     public void testHandleExceptinoOnSendSendFiles() throws Throwable {
```

Comment:
s/Exceptino/Exception/ while you are here?

Reply:
Sounds like a worthwhile correctino. Pushed pushed f4759cd8fc, fixing the send send files too.